music21/_version.py

```diff
@@ -42,7 +42,7 @@
 Changing this number invalidates old pickles -- do it if the old pickles create a problem.
 '''
 
-__version_info__ = (7, 0, 7)  # can be 4-tuple: (7, 0, 5, 'a2')
+__version_info__ = (7, 0, 7, 'b1')  # can be 4-tuple: (7, 0, 5, 'a2')
```

Comment:
will need to be changed to merge w/ other recent merges.

Reply:
During Alpha/Beta phases, it's okay not to update this immediately.  Only when released.  Or you know it'll break everything for everyone.

Reply:
Yes and this change would actually take the version number backward, so needs to be backed out.